### DISCUSSION TITLE
米家空调伴侣2 添加用电量信息

### DISCUSSION THREAD

**Discussion by vate34** [XiaoMi/ha_xiaomi_home#181] [Ideas]
米家空调伴侣2（lumi.acpartner.mcn02）传感器部分仅包含了当前电功率参数，未包含本日电量和本月电量，希望未来能够添加上
![截屏2024-12-18 23 09 14](https://github.com/user-attachments/assets/ef0bf355-83c8-48ff-8ba4-4b4332b86a77)

**Comment by JimmyKmi** [comment]
补充信息：这玩意没有用电量数据的spec，看似是另外在云端作统计拉取的，参考
https://mijia.wiki/products/lumi.acpartner.mcn02

**Reply by lzcapp** [reply]
我查了也是没有用电量，不知道第三方的插件是从哪里查的还是它自己算的。
![image](https://github.com/user-attachments/assets/fa68daf6-5c0a-4d5b-9740-223b3a8fff21)
